lib/tasks/user.rake

```diff
@@ -1,7 +1,7 @@
 namespace :user do
   task prepare_leave_for_new_year: :environment do
-  	users = User.all
-  	users.each do |user|
+    users = User.all
+    users.each do |user|
```

Comment:
Von apidock zu find_each: Note: This method is only intended to use for batch processing of large amounts of records that wouldn’t fit in memory all at once. If you just need to loop over less than 1000 records, it’s probably better just to use the regular find methods.

Klingt so als wäre User.all.each do |user| die bessere Variante. Eine eigene Variable Users braucht man allerdings nicht.